flybywiresim/aircraft
# fix: decals update for the new sim

Fixes #[issue_no]

## Summary of Changes

Should fix some missing decals in the A380X within the new Sim

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Load in MSFS2024, check that the overhead panel has all the decals and the rest of the cockpit.
1. Load in MSFS2020, check that the overhead panel has all the decals and the rest of the cockpit. NOthing should have changed.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
